### PR TITLE
test(e2e): add connector bytecode update admin test

### PR DESCRIPTION
**Pull request by lumtis**
# Description

Add an admin test testing ZEVM connector bytecode update

Closes: https://github.com/zeta-chain/node/issues/1999

Execution example: https://github.com/zeta-chain/node/actions/runs/8644743022/job/23700500386
(the liquidity cap test fails, issue tracked for this one)

### CODE REVIEW

File: cmd/zetae2e/local/local.go [hunk 1]
```diff
@@ -286,8 +286,10 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 	}
 	if testAdmin {
 		eg.Go(adminTestRoutine(conf, deployerRunner, verbose,
+			e2etests.TestZetaWithdrawName,
```
Comment (ws4charlie) [inline cmd/zetae2e/local/local.go:289]: why is this `TestZetaWithdrawName` required here?

Comment (lumtis) [reply cmd/zetae2e/local/local.go:289]: Nice catch, this is a line I added for testing purposes, I forgot to remove

File: e2e/e2etests/test_update_bytecode_connector.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	if err != nil {
+		panic(err)
+	}
+	r.Logger.Info("New contract code hash: %s", codeHashRes.CodeHash)
+
+	r.Logger.Info("Updating the bytecode of the Connector")
+	msg := fungibletypes.NewMsgUpdateContractBytecode(
+		r.ZetaTxServer.GetAccountAddress(0),
+		r.ConnectorZEVMAddr.Hex(),
+		codeHashRes.CodeHash,
```
Comment (ws4charlie) [inline e2e/e2etests/test_update_bytecode_connector.go:45]: Just to understand. The idea is to 1stly write expected byte code to zEVM by `DeployTestZetaConnectorZEVM` and then replace existing connector account's `CodeHash` in order to re-point underlying evm account's code to the newly deployed code. The only useful part of newly deployed contract is its byte code; the contract self (e.g. address) will be just forgotten?

Comment (lumtis) [reply e2e/e2etests/test_update_bytecode_connector.go:45]: >The only useful part of newly deployed contract is its byte code; the contract self (e.g. address) will be just forgotten?

Yes, correct

File: cmd/zetae2e/local/local.go (hunk 1 above)
Comment (lumtis) [reply cmd/zetae2e/local/local.go]: 9297e265

**Review by ws4charlie — APPROVED** [review]
lgtm

**Pull request merged**